modules/shared/src/main/scala/org/tesselation/schema/trust.scala

```diff
@@ -30,7 +31,14 @@ object trust {
   }
 
   @derive(decoder, encoder, show)
-  @newtype
-  case class PublicTrust(labels: Map[PeerId, Double])
+  case class PublicTrust(
+    labels: Map[PeerId, Double],
+    /**
+      * Randomization is only required in the event of gossiped and signed data to prevent
+      * reversions from being filtered out. When requested from a node directly it is
+      * unnecessary.
+      */
```

Comment:
Can you explain it more? It sounds it makes no sense. Gossip and direct request are interchangeable and transparent here in that layer.

Reply:
@ryle-goehausen I think after recent addition of counters by @marcinwadon it's not necessary. Spread messages are unique because counter is unique so you can spread twice the same content if you need to do so.

Reply:
As long as you guys can verify removing this is fine, that's fine with me. I remember in earlier comments it was mentioned that you're filtering repeats by recently seen hashes -- but if you do the counter comparison before the filter stage I can remove this -- or if it doesn't actually impact it that works. It seems in code like the hash is being computed and used without reference to the counter, but if that doesn't matter then I can just remove it.

Reply:
Give it a try and check on cluster. IMO it will produce different rumor hashes, and there is a counter as well.

Reply:
Removed this but had to leave it as kryo class due to rumor type signature:

>/Users/ryle/projects/tessellation/modules/core/src/main/scala/org/tesselation/domain/cluster/programs/TrustPush.scala:29:19
inferred type arguments [org.tesselation.schema.trust.PublicTrust] do not conform to method spread's type parameter bounds [A <: AnyRef]
      _ <- gossip.spread(trust)

Doesn't with work @newtype